hopsworks-common/src/main/java/io/hops/hopsworks/common/hive/HiveController.java

```diff
@@ -176,7 +194,12 @@ public void createDatabase(Project project, Users user, DistributedFileSystemOps
       dfso.setPermission(dbPath, fsPermission);
 
       // Set the default quota
-      dfso.setHdfsSpaceQuotaInMBs(dbPath, settings.getHiveDbDefaultQuota());
+      switch (datasetType) {
+        case HIVEDB:
```

Comment:
@Limmen break and default?

Reply:
Good catch, do we need a default? It would be for throwing an exception, should I add it?. We only have two types of valid hive database types. I don't think a hive database dataset should be allowed to have the dataset type "DATASET" since we have "HIVEDB" and "FEATURESTOREDB" for these particular datasets.

Reply:
Ok fixed, an exception is thrown at the beginning of the method so we do not need a default case